src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/RespondentTellSomethingElseController.java

```diff
@@ -85,47 +82,10 @@ public ResponseEntity<CCDCallbackResponse> aboutToStartRespondentTSE(
             return ResponseEntity.status(FORBIDDEN.value()).build();
         }
         CaseData caseData = ccdRequest.getCaseDetails().getCaseData();
-        if (Helper.isClaimantNonSystemUser(caseData)) {
-            caseData.setResTseNotAvailableWarning(YES);
-        }
 
         return getCallbackRespEntityNoErrors(caseData);
     }
 
-    /**
-     * For displaying error and preventing user to proceed to the event, if
-     * the other party (Claimant) is a non-system user.
-     *
-     * @param ccdRequest holds the request and case data
-     * @param userToken  used for authorization
-     * @return Callback response entity with confirmation header and body
-     */
-    @PostMapping(value = "/showError", consumes = APPLICATION_JSON_VALUE)
-    @ApiResponses(value = {
-        @ApiResponse(responseCode = "200", description = "Accessed successfully",
-                content = {
-                    @Content(mediaType = "application/json",
-                            schema = @Schema(implementation = CCDCallbackResponse.class))
-                }),
-        @ApiResponse(responseCode = "400", description = "Bad Request"),
-        @ApiResponse(responseCode = "500", description = "Internal Server Error")
-    })
-    public ResponseEntity<CCDCallbackResponse> showError(

```

Comment:
does it make sense to keep this method (just remove the line about non system users?). Means it can be reused later if other errors need to be handled?